v0/sources/auth0/util.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * This function returns the organization_id from apiUrl path
+ * example
+    path : "/api/v2/organizations/org_DqzHLxgABzej3mwL/members"
+    groupId : org_DqzHLxgABzej3mwL
+ * @param {*} event
+ */
+const getGroupId = event => {
+  const { path } = event.details.request;
+  const pathArray = path.split("/");
+  return pathArray[pathArray.length - 2];
```

Comment:
What if any of the fields in the path `event.details.request.path` is undefined?

Reply:
discussed this with @shrouti1507 and the final outcome is we will drop event if path is not there, i made changes according to that